Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -318,6 +318,38 @@ void PresentAlert(AlertArguments arguments)
 		{
 			var window = new UIWindow { BackgroundColor = Color.Transparent.ToUIColor() };
 
+			if (arguments.Visual.IsMaterial())
+			{
+				var malert = AlertController.Create(arguments.Title, arguments.Message);
+
+				if (arguments.Cancel != null)
+				{
+					malert.AddAction(AlertAction.Create(arguments.Cancel, ActionEmphasis.Low, act => {
+						window.Hidden = true;
+						arguments.SetResult(false);
+					}));
+				}
+
+				if (arguments.Accept != null)
+				{
+					malert.AddAction(AlertAction.Create(arguments.Accept, ActionEmphasis.Low, act => {
+						window.Hidden = true;
+						arguments.SetResult(true);
+					}));
+				}
+
+				var scheme = new AlertScheme();
+				AlertControllerThemer.ApplyScheme(scheme, malert);
+
+				window.RootViewController = new UIViewController();
+				window.RootViewController.View.BackgroundColor = Color.Transparent.ToUIColor();
+				window.WindowLevel = UIWindowLevel.Alert + 1;
+				window.MakeKeyAndVisible();
+				window.RootViewController.PresentViewController(malert, true, null);
+
+				return;
+			}
```

Comment:
This is just for material alerts on iOS, and is a hack right now. Do not ever write code like this!